src/ProtocolFees.sol

```diff
@@ -53,6 +54,9 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
     {
         if (msg.sender != address(protocolFeeController)) InvalidCaller.selector.revertWith();
         if (_isUnlocked()) ContractUnlocked.selector.revertWith();
```

Comment:
this `_isUnlocked` check can also be safely removed; it was similarly added in #856. I'm indifferent about this though.

Reply:
[0c762cb](https://github.com/Uniswap/v4-core/pull/886/commits/0c762cb11d52f29cef474e7e6fc44cc74a395f61)